whipper/command/cd.py

```diff
@@ -341,7 +350,19 @@ def doCommand(self):
         else:
             logger.info("creating output directory %s", dirname)
             os.makedirs(dirname)
-
+        self.coverArtPath = None
+        if (self.options.fetch_cover_art in {"embed", "complete"} and
+                importlib.util.find_spec("PIL") is None):
+            logger.warning("the cover art option '%s' won't be honored "
+                           "because the 'pillow' module isn't available",
+                           self.options.fetch_cover_art)
+        elif self.options.fetch_cover_art in {"file", "embed", "complete"}:
```

Comment:
Dict seems entirely unreasonable here… ?

```suggestion
        elif self.options.fetch_cover_art in ("file", "embed", "complete"):
```

Reply:
It isn't a `dict`, it's a `set`:

```python
>>> type({'file', 'embed', 'complete'})
<class 'set'>